src/main/java/com/github/brainlag/nsq/Connection.java

```diff
@@ -39,6 +40,9 @@
     private final EventLoopGroup eventLoopGroup;
     private final NSQConfig config;
 
+    public static final long HEARTBEAT_MAX_INTERVAL = 1L*60L*1000L;//default one minute
```

Comment:
Maybe HEARTBEAT_MAX_INTERVAL should be set relative to `heartbeatInterval` in `NSQConfig`?